ci-operator/step-registry/telco5g/cluster-setup/telco5g-cluster-setup-commands.sh

```diff
@@ -232,9 +242,44 @@ cat << EOF > ~/fetch-information.yml
     shell: kcli ssh root@${CLUSTER_NAME}-installer 'oc get node'
 EOF
 
-status=0
-ANSIBLE_STDOUT_CALLBACK=debug ansible-playbook -i $SHARED_DIR/inventory ~/ocp-install.yml -vv || status=$?
-ansible-playbook -i $SHARED_DIR/inventory ~/fetch-kubeconfig.yml -vv || true
-ANSIBLE_STDOUT_CALLBACK=debug ansible-playbook -i $SHARED_DIR/inventory ~/fetch-information.yml -vv || true
+cat << EOF > ~/test-jq.sh
+  #!/bin/bash
+  # simple bash script to check if cluster is available
+  oc get clusterversion -o json|jq '.items[0].status.conditions[]|select(.type=="Available").status'
```

Comment:
well, there is definitely no `jq` on the installer, you need to install it firstly.

Reply:
Why just not something like `kcli ssh root@cnfdd3-installer 'oc get clusterversion'  | awk {'print $3'} | tail -1`

Reply:
When using `kcli ssh root@cnfdd3-installer 'oc get clusterversion'  | awk {'print $3'} | tail -1` it treats the empty 'version' column as a separator, and `$3` ends up being the 'True' from 'Progressing' instead of the 'False' from 'Available'... this means that whether the cluster is available or not, this particular check would return 'True' in a lot of cases where it would not be desirable.

This is initially why I opted to parse the json, since human-readable output can be hard to deal with and the structure of the json output is more predictable.

However, I've found a way to do this without `jq` -- using the -o jsonpath flag for the oc command.